draft-ietf-core-multipart-ct.md

```diff
@@ -55,7 +55,7 @@ informative:
 
 This memo defines application/multipart-core, an
 application-independent media-type that can be used
-to combine representations of several different media types into a single
+to combine representations of different media types into a single
```

Comment:
I would be even more explicit and say "zero or more different media types" in these three places.

Reply:
done in 23724fd